test/test-cmake.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 Huang Qinjin
+#
+# Permission to use, copy, modify, and/or distribute this software for any
+# purpose with or without fee is hereby granted, provided that the above
+# copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+# WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+# ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+# WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+# ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+# OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+. "${0%/*}/test.sh"
+
+
+CMAKE_ARGS=(
+    -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    -DCMAKE_SYSTEM_NAME=Windows
+)
+
+case $OSTYPE in
+    darwin*)
+        CMAKE_ARGS+=(
+            -DCMAKE_EXE_LINKER_FLAGS=/MANIFEST:NO
+            -DCMAKE_SHARED_LINKER_FLAGS=/MANIFEST:NO
+            -DCMAKE_MSVC_DEBUG_INFORMATION_FORMAT=Embedded
```

Comment:
Why are these arguments needed on Darwin? I presume that means that msvctricks.exe doesn't work there (breaking the mt.exe return code rewrite, and leaving cmake/ninja waiting on the mspdbsrv background process)? A comment would probably be good.

Reply:
https://github.com/mstorsjo/msvc-wine/pull/65#issuecomment-1531747302

Reply:
Oh, ok, thanks for reminding me! A comment mentioning these reasons would be good.

Reply:
Done.